src/Mul.php

```diff
@@ -0,0 +1,24 @@
+<?php
+namespace Laminas\Filter;
+
+use Laminas\Filter\AbstractFilter;
+use Laminas\Filter\Exception\InvalidArgumentException;
+
+class Mul extends AbstractArithmeticOperation
+{
+    /**
+     *
+     * {@inheritdoc}
+     * @see \Laminas\Filter\FilterInterface::filter()
+     */
+    public function filter($value)
+    {
+        if (! isset($this->options['operand'])) {
+            throw new InvalidArgumentException(sprintf('%s expects argument value; received none', __METHOD__));
+        }
+
+        $value = (float) $value;
+        $operand = (float) $this->options['operand'];
+        return ($value * $operand);
```

Comment:
Same remark as for `Add` and `Sub`: if integers are used, I'd expect an integer return value, not casting to float.